specs/tldr-lint-helper.js

```diff
@@ -26,15 +26,17 @@ containsOnlyErrors = function(errors, expected) {
     expected = Array.prototype.splice.call(arguments, 1);
   }
   expected.forEach(function(error) {
-    if (!containsErrors(errors, error)) { 
+    if (!containsErrors(errors, error)) {
       console.error('Couldnt find error', error, 'in these errors')
       console.error(errors)
       return false;
     };
   });
-  for (error of errors) {
-    if (!expected.some(function(expectedCode) { 
-      return error.code === expectedCode; 
+  var error;
```

Comment:
It is not that redundant, if you follow Douglas Crockford's recommendations. But ok, i'll remove it.